src/linear_algebra.jl

```diff
@@ -98,13 +107,6 @@ function _solve(A::AbstractMatrix, b::AbstractArray, do_simplify)
     do_simplify ? SymbolicUtils.simplify.(sol, expand=true) : sol
 end
```

Comment:
So this PR only helps when you have to solve_for one variable? What was the case where just calling this with 1-element arrays wouldn't work?

Reply:
It could work, but if someone solves a scalar equation, one should use the scalar version. I could make a special case for vectors, too.